spec/components/schemas/reports/message/entry/base.ts

```diff
@@ -0,0 +1,43 @@
+// tslint:disable:max-line-length
+import { SchemaObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../../../utils/ref';
+
+const base: SchemaObject = {
+  title: 'Entry object for Message Report',
```

Comment:
Aqui também igual abaixo: "The entry object for Message Report resource." seria minha sugestão (com ponto final kkkk).